samples/compensation-json/compensation.sw.json

```diff
@@ -0,0 +1,75 @@
+{
+  "id": "compensation",
+  "version": "1.0",
+  "specVersion": "0.8",
+  "name": "Workflow Error example",
```

Comment:
The capitalization of those words is ok?

Reply:
I kept everything from kogito-examples repo. Should we change it to something else?